ros_cross_compile/docker_client.py

```diff
@@ -18,6 +18,8 @@
 
 import docker
 
+DEFAULT_COLCON_DEFAULTS_FILE = 'defaults.yaml'
```

Comment:
Should this be passed by the used somewhere? Where is this specified?

Reply:
It's the default name of the defaults file in colcon, but I'm using the explicit file variable rather than just setting COLCON_HOME to the workspace. Maybe that would be better. We could also expose it to the user to specify. You think I should go for that instead?

Reply:
I see, I assumed the purpose of this PR was to enable the user specify the file from the CLI but this also works.